src/docs/resources/faq.md

```diff
@@ -401,7 +401,7 @@ is retained after a reload. This means you can quickly iterate
 on a screen deeply nested in your app, without starting
 from the home screen after every reload.
 
-### How is 'hot reload' different from 'hot restart'?
+### How is `hot reload` different from `hot restart`?
```

Comment:
Code font?!!! I'd personally use italics or nothing. (The font doesn't show up in the TOC, fwiw.)